libs/utils/android/workloads/uibench.py

```diff
@@ -84,6 +84,11 @@ def run(self, out_dir, test_name, duration_s, collect=''):
         # Keep track of mandatory parameters
         self.out_dir = out_dir
         self.collect = collect
+        # For systrace collection, the duration is expected to be part of `collect` parameter
+        # This is because systrace capturing currently doesnt support CTRL+C stopping
+        # so tracingStop() call just waits instead.
+        if collect == 'systrace':
+            self.collect = 'systrace_' + str(duration_s)
```

Comment:
Since the API is already broken, i.e. systrace expect the duration to be defined as part of the `capture` string, should not be better to just remove this non-sense `duration_s` parameter and encode the duration in the `systrace_DURATION` parameter from the caller itself.

This would make the current code more aligned with the new API once we have the proper "on demand" stop support (see #331)